ansible/configure-hosts.yml

```diff
@@ -1,6 +1,8 @@
 ---
 - import_playbook: fix-homedir-ownership.yml
   tags: fix-homedir
+- import_playbook: growroot.yml
+  tags: growroot
```

Comment:
The SKC growroot.yml playbook is added as a pre-host configure hook. Is that too late?

Reply:
That should only affect the ansible control host. Should we target the new playbook there instead?

Reply:
I'll probably redeploy my multinode overnight so I'll try it then

Reply:
All good. Done